test/testdata/infer/fields.rb.cfg-text.exp

```diff
@@ -20,11 +20,11 @@ method ::Foo#initialize {
 bb0[rubyRegionId=0, firstDead=9]():
     @ivar$3: Integer = alias @ivar
     <self>: Foo = cast(<self>: NilClass, Foo);
-    <cfgAlias>$6: T.class_of(Sorbet::Private::Static) = alias <C Static>
-    <cfgAlias>$8: T.class_of(Integer) = alias <C Integer>
-    <statTemp>$4: Sorbet::Private::Static::Void = <cfgAlias>$6: T.class_of(Sorbet::Private::Static).keep_for_typechecking(<cfgAlias>$8: T.class_of(Integer))
-    <castTemp>$9: Integer(0) = 0
-    @ivar$3: Integer = cast(<castTemp>$9: Integer(0), Integer);
+    <cfgAlias>$5: T.class_of(Integer) = alias <C Integer>
+    keep_for_ide$4: T.class_of(Integer) = <cfgAlias>$5
+    keep_for_ide$4: T.untyped = <volatile-read> keep_for_ide$4
```

Comment:
related to the comment below, it looks like we're re-assigning this variable and squashing the type?